onnxruntime/contrib_ops/cpu/bert/attention_common.h

```diff
@@ -114,7 +114,8 @@ struct GroupQueryAttentionParameters {
   int local_window_size;
   bool kv_share_buffer;
   bool is_packed_qkv;
-  bool is_prompt;  // determines if seqlens_k is past or kv sequence length tensor
+  bool is_interactive;  // indicates whether we have past context and seqlen > 1
```

Comment:
>is_interactive [](http://example.com/codeflow?start=7&length=14)

nit: change is_prompt to is_first_prompt and is_interactive to is_subsequent_prompt